packages/core/src/renderer/components/custom-resources/crd-resources.tsx

```diff
@@ -106,7 +106,11 @@ class NonInjectedCustomResources extends React.Component<Dependencies> {
             isNamespaced && (
               <NamespaceSelectBadge namespace={customResource.getNs() as string} />
             ),
-            ...extraColumns.map((column) => safeJSONPathValue(customResource, column.jsonPath)),
+            ...(
+              extraColumns
+                .map((column) => safeJSONPathValue(customResource, column.jsonPath))
+                .map(formatJSONValue)
+            ),
```

Comment:
Is this the real fix? In this type of PRs it would be easier if there was a separate commit for consolidating types and the real fix.